data/dialog.yaml

```
@@ -153,28 +153,32 @@ Gerud Ancient Dungeon Reward:
 Found Dog Mitchell:
   conversation:
     - label 1:
-      - print(While you walk into the Los Gerudos grand avenue, and admire the sea, visible from a long distance, you hear a noise, coming from behind you. You slowly turn behind. You find a dog; he must have followed you since you've left that little juwelery store. He must have been lost by his dear owners!)
+      - print(While you walk into the Los Gerudos grand avenue, and admire the sea, visible from a long distance, you hear a noise, coming from behind you. You slowly turn behind. You find a dog; he must have followed you since you've left that little jewelry store. He must have been lost by his dear owners!)
       - print(You decide to take a look...)
       - wait(2)
       - "print(You found a golden collar on his neck, with inscribed letters on it. It says: “Mitchell - property of the Coopers; Please return this dog if you find him at the 13 Yasward Drive, Los Gerudos“. The dog looks at you with big eyes, seeming like he wants you to take him back to its owners.\n)"
-  use actions: false
+  use actions: true
+  actions:
+    add to diary:
+      known npcs:
+      - Marry Joselina
 
 Given Back Dog Mitchell:
   conversation:
     - label 1:
       - print(You finally arrive to the 13 Yasward Drive, Los Gerudos, the village of the Coopers. You knock at the door.)
       - wait(2)
-      - print(A bulter finally opens the great doors, and greets you, before asking who you are and what do you want. You explain to him everything, before showing him the dog. Suddently, the bulter shouts:)
-      - print(“Mrs. Cooper! We've found Mitchell back!“\n)
+      - print(A bulter finally opens the great doors, and greets you, before asking who you are and what do you want. You explain to him everything, before showing him the dog. Suddenly, the bulter shouts:)
+      - print(“- Mrs. Cooper! We've found Mitchell back!“\n)
```

```suggestion
      - print(“- Mrs. Cooper! We've found Mitchell again!“\n)
```
```suggestion
      - print(“- Mrs. Cooper! Mitchell is back!“\n)
```
Not sure which you're going for here.